build/cache/.gitignore

```diff
@@ -0,0 +1 @@
+container.bin
```

Comment:
I've always thought that you should put a ```.gitkeep``` in an empty directory you want to have in the repo and then have one ```.gitignore``` at the root of the codebase with the full path to any file you want to ignore.

Your code base though, so if you want lots of ```.gitignore``` files this is fine :smile: